en/docs/learn/configuring-access-delegation-with-oauth2.md

```diff
@@ -0,0 +1,60 @@
+# Configuring Access Delegation with OAuth 2.0
+
+### Introduction
```

Comment:
## Make changes in other places as well.

Reply:
Fixed with 
aeb5e16